velox/core/QueryCtx.cpp

```diff
@@ -79,4 +79,16 @@ QueryCtx::QueryCtx(
   return fmt::format("query.{}.{}", queryId.c_str(), seqNum++);
 }
 
+void QueryCtx::checkNumSpillBytes(uint64_t bytes) {
+  if (queryConfig_.maxSpillBytes() > 0) {
```

Comment:
We might want to update spill bytes no matter there is a limit enforcement or not? Thanks!